src/app/Main/Body/QueryPage/QueryPage-Container.component.tsx

```diff
@@ -263,41 +270,48 @@ export function QueryPageContainerComponent(): JSX.Element {
         const newUniqueValues = _.cloneDeep(uniqueDataValues);
         const filterStatusList: number[] = [];
         const getConditionalFilterValues = filter.displayedFilters.map(
-            async (mainFilter) => {
-                const selectedFilter = filter.selectedFilter[mainFilter];
-                if (_.isEmpty(selectedFilter)) {
-                    const conditionalFilterUrl = `${FILTER_URL}/${mainFilter}/${history.location.search}`;
+            async (displayedFilter) => {
+                const urlParams = new URLSearchParams(history.location.search);
+                urlParams.delete("row");
+                urlParams.delete("column");
+                urlParams.delete(displayedFilter);

```

Comment:
This is for fixing the api issue you mentioned in your deleted comment?

Reply:
Yes